.travis.yml

```diff
@@ -9,6 +9,7 @@ go:
     - 1.9.x
     - 1.10.x
     - 1.11.x
+    - 1.12.x
```

Comment:
ah looks like the current build fails (https://travis-ci.org/aws/amazon-ecs-cli/jobs/511105968) because the env variable is missing 😝

Looks like we need something like:
```yaml
env:
  - GOCACHE=$HOME/.cache/go-build
```

Reply:
lol , i will fix asap

Reply:
@efekarakus build fixed!